LPS-79534 Adding test

4lejandrito opened the pull request.

4lejandrito committed:
LPS-79534 Escaping subject before setting it as body

4lejandrito committed:
LPS-79534 Adding test

liferay-continuous-integration commented:
CI is automatically triggering &quot;ci:test:sf&quot; for this pull to run Source Formatter tests.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

4lejandrito commented:
ci:test

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/2179/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-8.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2179/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/2180/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-8.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2180/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 2 out of 2 jobs passed in 13 minutes 12 seconds 858 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d5b6199697ec0aae802146cebe43caf07d3244b6">d5b6199697ec0aae802146cebe43caf07d3244b6</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/c5f9d407a6b03bd795e4cc86bb17cdae63d32e21">c5f9d407a6b03bd795e4cc86bb17cdae63d32e21</a></p>2 out of 2 jobs PASSED<details><summary><strong>2 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/2179/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/180169/">test-portal-acceptance-pullrequest-batch(master)/source-format-current-branch-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2179/jenkins-report.html">here</a>.</h5></details></html>

4lejandrito committed:
LPS-77556 Adjusting size to be 16x16 as required by the designs

This is because clay does not set the proper font size to icon-only
buttons. When they fix it we will simplify our css.

liferay-continuous-integration commented:
Closing and locking pull request because pull requests sent to this user may not be updated. Please resend this pull request.

liferay-continuous-integration commented:
<h1>The pull request tester is still running.</h1><p>Please wait until you get the <i><b>final report</b></i> before running 'ci:retest'.</p><p>See this link to check on the status of your test:</p><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/2180/">test-portal-acceptance-pullrequest(master)</a></li></ul><p>@4lejandrito</p><hr /><h1>However, the pull request was closed.</h1><p>The pull request was closed because the following critical batches had failed:</p><ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/216210/">unit-jdk8</a></li></ul><p>For information as to why we automatically close out certain pull requests see this <a href="https://in.liferay.com/web/global.engineering/wiki/-/wiki/Quality+Assurance+Main/Test+Batch+Automatic+Close+List">article</a>.</p><p auto-close="false"><strong><em>*This pull will no longer automatically close if this comment is available. If you believe this is a mistake please re-open this pull by entering the following command as a comment.</em></strong></p><pre>ci&#58;reopen</pre><hr /><h3>Critical Failure Details:</h3><div><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/216210/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></h5><div><h6>Job Results:</h6><p>2559 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/216210//consoleText">AXIS_VARIABLE=0,label_exp=!master #216210</a><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/216210//testReport/com.liferay.portal.kernel.util/UnicodePropertiesTest/testSetBlankProperty">UnicodePropertiesTest.testSetBlankProperty</a><pre><code>junit.framework.AssertionFailedError: expected:&lt;[key1=
key3=null
]&gt; but was:&lt;[]&gt;
	at com.liferay.portal.kernel.util.UnicodePropertiesTest.testSetBlankProperty(UnicodePropertiesTest.java:35)
</code></pre></div></li></ol></div></li></ol></div>

liferay-continuous-integration commented:
<html><h3>:x: ci:test - 95 out of 114 jobs passed in 2 hours 54 minutes 55 seconds 137 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/d5b6199697ec0aae802146cebe43caf07d3244b6">d5b6199697ec0aae802146cebe43caf07d3244b6</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/c5f9d407a6b03bd795e4cc86bb17cdae63d32e21">c5f9d407a6b03bd795e4cc86bb17cdae63d32e21</a></p>95 out of 114 jobs PASSED<div><h4>19 Failed Jobs:</h4><ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest(master)/2180/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214474/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/2</strong></strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221494/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/4</strong></strike></a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76364/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/9</strong></strike></a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152850/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mariadb100-jdk8/4</strong></strike></a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152854/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/0</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/216209/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/integration-oracle121-jdk8</strong></strike></a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208714/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</strong></strike></a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152855/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-db2105-jdk8</strong></strike></a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76369/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk8</strong></strike></a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201419/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mariadb100-jdk8</strong></strike></a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/230979/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk8</strong></strike></a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228145/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-oracle121-jdk8</strong></strike></a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/145204/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk8</strong></strike></a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208713/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-sybase160-jdk8</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214478/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221925/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/portal-frontend-js-jdk8</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/216210/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</strong></strike></a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/195226/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/wsdd-builder-jdk8</strong></strike></a></li></ul></div><details><summary><strong>95 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76363/">test-portal-acceptance-pullrequest-batch(master)/bundle-blacklist-restart-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157517/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76362/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221492/">test-portal-acceptance-pullrequest-batch(master)/empty-temp-dir-jdk8</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157518/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/206254/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tcserver32-mysql56-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221493/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-db2105-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214473/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mariadb100-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221918/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql56-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/206255/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-oracle121-jdk8/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157519/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-postgresql94-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208705/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-sybase160-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228139/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-weblogic121-mysql56-jdk8/0</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208706/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-websphere85-mysql56-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/206256/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-wildfly100-mariadb100-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/206257/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228140/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221919/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/10</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/230974/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/11</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208709/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/12</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208932/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/13</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/206258/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/14</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228142/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/15</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76366/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/16</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157522/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/17</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214475/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/18</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228141/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/19</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221495/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/20</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157521/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/21</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/230975/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/22</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76367/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/23</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208934/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/24</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/195223/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/25</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201414/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/26</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157523/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/27</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221921/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/28</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152849/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/29</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208708/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/3</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208933/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/30</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75753/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/31</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76365/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/5</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208707/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/6</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157520/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/7</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221920/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/216208/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/216207/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75754/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/2</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/195225/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/3</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/195224/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-jboss70-mysql56-jdk8/4</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152851/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mariadb100-jdk8/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/180206/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mariadb100-jdk8/1</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/180207/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mariadb100-jdk8/2</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208711/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mariadb100-jdk8/3</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/215621/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql56-jdk8/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228226/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql56-jdk8/1</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228225/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql56-jdk8/2</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75755/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql56-jdk8/3</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/215620/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql56-jdk8/4</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/206261/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-oracle121-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221496/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-oracle121-jdk8/1</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/206260/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-oracle121-jdk8/2</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/206259/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-oracle121-jdk8/3</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214476/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-oracle121-jdk8/4</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157524/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-postgresql94-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221922/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-postgresql94-jdk8/1</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221497/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-postgresql94-jdk8/2</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157525/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-postgresql94-jdk8/3</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208935/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-postgresql94-jdk8/4</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152853/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/1</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201418/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/2</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152852/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/3</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208712/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/4</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228144/">test-portal-acceptance-pullrequest-batch(master)/integration-db2105-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/230977/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76368/">test-portal-acceptance-pullrequest-batch(master)/integration-mariadb100-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201417/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201416/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221923/">test-portal-acceptance-pullrequest-batch(master)/integration-sybase160-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214477/">test-portal-acceptance-pullrequest-batch(master)/javadoc-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/230978/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152856/">test-portal-acceptance-pullrequest-batch(master)/lpkg-override-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76370/">test-portal-acceptance-pullrequest-batch(master)/lpkg-persistence-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221499/">test-portal-acceptance-pullrequest-batch(master)/lpkg-release-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/228143/">test-portal-acceptance-pullrequest-batch(master)/lpkg-startup-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221498/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk8</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/216211/">test-portal-acceptance-pullrequest-batch(master)/portal-license-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/208936/">test-portal-acceptance-pullrequest-batch(master)/portal-startup-space-in-path-jdk8</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/195227/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221924/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/75756/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201420/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201565/">test-portal-acceptance-pullrequest-batch(master)/slim-profile-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152857/">test-portal-acceptance-pullrequest-batch(master)/source-format-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/230980/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/865/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-8.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2180/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/214474/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/2</a></h5><div><h6>Job Results:</h6><p>19 Tests Passed.<br/>5 Tests Failed.</p></div><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/214474//consoleText">AXIS_VARIABLE=0,label_exp=!master #214474</a><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/214474//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PGTags_AddSelectTagViaBlogs_">LocalFile.PGTags#AddSelectTagViaBlogs</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/0/LocalFile.PGTags_AddSelectTagViaBlogs/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/0/LocalFile.PGTags_AddSelectTagViaBlogs/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/214474//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PGTags_AddTagViaBlogs_">LocalFile.PGTags#AddTagViaBlogs</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/0/LocalFile.PGTags_AddTagViaBlogs/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/0/LocalFile.PGTags_AddTagViaBlogs/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/214474//consoleText">AXIS_VARIABLE=1,label_exp=!master #214474</a><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/214474//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PGTags_ViewTagCloudIncrease_">LocalFile.PGTags#ViewTagCloudIncrease</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/1/LocalFile.PGTags_ViewTagCloudIncrease/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/1/LocalFile.PGTags_ViewTagCloudIncrease/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/1/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/214474//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PGWiki_EditWikiFrontPage_">LocalFile.PGWiki#EditWikiFrontPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/1/LocalFile.PGWiki_EditWikiFrontPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/1/LocalFile.PGWiki_EditWikiFrontPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/214474//consoleText">AXIS_VARIABLE=2,label_exp=!master #214474</a><ol><li><div><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/214474//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_TagsAdmin_EditTagAndSave_">LocalFile.TagsAdmin#EditTagAndSave</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/2/LocalFile.TagsAdmin_EditTagAndSave/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/2/LocalFile.TagsAdmin_EditTagAndSave/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/2/2/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/221494/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/4</a></h5><div><h6>Job Results:</h6><p>17 Tests Passed.<br/>3 Tests Failed.</p></div><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/221494//consoleText">AXIS_VARIABLE=0,label_exp=!master #221494</a><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/221494//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DocumentsadministrationUsecase_AddDocumentTypeAndMetadata_">LocalFile.DocumentsadministrationUsecase#AddDocumentTypeAndMetadata</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/0/LocalFile.DocumentsadministrationUsecase_AddDocumentTypeAndMetadata/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/0/LocalFile.DocumentsadministrationUsecase_AddDocumentTypeAndMetadata/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/221494//consoleText">AXIS_VARIABLE=1,label_exp=!master #221494</a><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/221494//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PGDocumentsandmedia_DeleteSubFolder_">LocalFile.PGDocumentsandmedia#DeleteSubFolder</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/1/LocalFile.PGDocumentsandmedia_DeleteSubFolder/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/1/LocalFile.PGDocumentsandmedia_DeleteSubFolder/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/1/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/221494//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PGDocumentsandmedia_EditDocumentType_">LocalFile.PGDocumentsandmedia#EditDocumentType</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/1/LocalFile.PGDocumentsandmedia_EditDocumentType/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/1/LocalFile.PGDocumentsandmedia_EditDocumentType/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/4/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/76364/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql56-jdk8/9</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/76364//consoleText">AXIS_VARIABLE=0,label_exp=!master #76364</a><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/76364//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_UIInfrastructureUsecase_RightToLeftSmoke_">LocalFile.UIInfrastructureUsecase#RightToLeftSmoke</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/9/0/LocalFile.UIInfrastructureUsecase_RightToLeftSmoke/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/9/0/LocalFile.UIInfrastructureUsecase_RightToLeftSmoke/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-tomcat90-mysql56-jdk8/9/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152850/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mariadb100-jdk8/4</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/152850//consoleText">AXIS_VARIABLE=0,label_exp=!master #152850</a><ol><li><div><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/152850//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmokeUpgrade_ViewPortalSmokeArchive6130_">LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchive6130</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-upgrade-tomcat90-mariadb100-jdk8/4/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive6130/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-upgrade-tomcat90-mariadb100-jdk8/4/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive6130/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-upgrade-tomcat90-mariadb100-jdk8/4/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/152854/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-wildfly100-mariadb100-jdk8/0</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/152854//consoleText">AXIS_VARIABLE=0,label_exp=!master #152854</a><ol><li><div><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/152854//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmokeUpgrade_ViewPortalSmokeArchive625_">LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchive625</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-upgrade-wildfly100-mariadb100-jdk8/0/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive625/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-upgrade-wildfly100-mariadb100-jdk8/0/0/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive625/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-8/1523449820418/test-portal-acceptance-pullrequest(master)/2180/functional-upgrade-wildfly100-mariadb100-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li>...</li></ol></details></html>
